common/flatpak-dir.c

```diff
@@ -1153,6 +1154,57 @@ flatpak_dir_remove_oci_files (FlatpakDir   *self,
   return TRUE;
 }
 
+static gchar *
+flatpak_dir_revokefs_fuse_create_mountpoint (const gchar *ref,
+                                             GError     **error)
+{
+  g_autoptr(GFile) cache_dir = NULL;
+  g_auto(GStrv) parts = NULL;
+  g_autofree gchar *cache_dir_path = NULL;
+  g_autofree gchar *mnt_dir = NULL;
+  g_autofree gchar *mountpoint = NULL;
+
+  cache_dir = flatpak_ensure_system_user_cache_dir_location (error);
+  if (cache_dir == NULL)
+    return NULL;
+
+  parts = flatpak_decompose_ref (ref, NULL);
```

Comment:
We should check for path == NULL and return an error here.  I realize this is unlikely to hit, but we already can return errors, so why not be extra careful.

Reply:
(and there is already an error out in flatpak_decompose_ref)

Reply:
handled error out case as suggested